LoopUI/Views/LoopStatusCircleView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  LoopStatusCircleView.swift
+//  LoopUI
+//
+//  Created by Arwain Karlin on 5/8/24.
+//  Copyright © 2024 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+public struct LoopStatusCircleView: View {
+    
+    public enum Status {
+        case closedLoopOn
+        case closedLoopOff
+        case closedLoopUnavailable
+        
+        var color: Color {
+            switch self {
+            case .closedLoopOn:
+                return .green  // Use guidanceColors
+            case .closedLoopOff:
+                return .red  // Use guidanceColors
+            case .closedLoopUnavailable:
+                return .orange // Use guidanceColors
```

Comment:
We'll want to use the environment to get the guidance colors here. Instead of these placeholders.